app/serializers/group_post_serializer.rb

```diff
@@ -5,20 +5,68 @@
 class GroupPostSerializer < ApplicationSerializer
   include PostItemExcerpt
 
-  attributes :id, :created_at, :title, :url, :category_id, :post_number, :topic_id, :post_type
+  attributes :id,
+             :created_at,
+             :topic_id,
+             :topic_title,
+             :topic_slug,
+             :topic_html_title,
+             :url,
+             :category_id,
+             :post_number,
+             :posts_count,
+             :post_type,
+             :username,
+             :name,
+             :avatar_template,
+             :user_title,
+             :primary_group_name
 
+  # TODO(keegan): Remove `embed: :object` after updating references in discourse-reactions
   has_one :user, serializer: GroupPostUserSerializer, embed: :object
   has_one :topic, serializer: BasicTopicSerializer, embed: :object
 
-  def title
+  def topic_title
     object.topic.title
   end
 
+  def topic_html_title
+    object.topic.fancy_title
+  end
```

Comment:
Do we call it `html_title` anywhere else? If not, I'd be tempted to stick with our normal 'fancy_title' naming for consistency.